sys/fbdev/gstg1fbdevsink.c

```diff
@@ -466,6 +491,16 @@ gst_g1_fbdevsink_class_init (GstG1FBDEVSinkClass * klass)
           "the image width must match the FB width.", DEFAULT_ZERO_MEMCPY,
           G_PARAM_READWRITE));
 
+  g_object_class_install_property (G_OBJECT_CLASS (klass), ARG_CX,
+      g_param_spec_int ("cx", "cx",
+          "offset of x in screen", -1, 800, DEFAULT_CX,
+          G_PARAM_READWRITE));
```

Comment:
Isn't it possible to remove this static dependency with screen size. For example, is it possible to retrieve these values from the framebuffer info...